JavaOpdracht6/familyTree/src/main/java/Person.java

```diff
@@ -148,6 +159,9 @@ public List<Person> getGrandChildren() {
 
     public void addPet(Person owner, Pet pet) {
         List<Pet> petsList = new ArrayList<>();
+        if (owner.getPets() != null) {
+            petsList.addAll(owner.getPets());
+        }
         petsList.add(pet);
         owner.setPets(petsList);
     }
```

Comment:
In declaratie: private List<Pet> pets = new ArrayList<>();
In methode: this.pets.add(pet);